oracle/Cargo.toml

```diff
@@ -14,6 +14,28 @@ ocaml = { version = "0.22.0", optional = true }
 rand = "0.7.3"
 rayon = { version = "1" }
 
-[features]
+# for tests
+serde = { version = "1.0", features = ["derive"] }
+serde_json = { version = "1.0" }
+hex = { version = "0.4" }
```

Comment:
nit: this is for export_test_vectors, if it was for tests then we would have put that in dev-dependencies

Reply:
These are required for running the tests
For example, `cargo test --test poseidon_tests` runs the tests in `poseidon_tests.rs`, which calls `serde_json::from_reader` then  `test_vectors -> hex_to_field -> hex::decode`

Reply:
>  I tried moving them to dev-dependencies before, but it was not happy. Feel free to update if you want to have a go at it.

that's because they are required for export_test_vectors which is not a test

Reply:
Quickly found a way to do it.

Reply:
that's cleaner, nice!